snapshot.go

```diff
@@ -13,9 +13,19 @@
 
 package firecracker
 
+import "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
+
 type SnapshotConfig struct {
 	MemFilePath         string
+	MemBackend          *models.MemoryBackend
 	SnapshotPath        string
 	EnableDiffSnapshots bool
 	ResumeVM            bool
 }
+
+func (cfg *SnapshotConfig) GetMemBackendPath() string {
```

Comment:
How about adding some comments to the function something like `if MemBackend is not set, then MemFilePath from SnapshotConfig will be returned` ?    
Just from the name of the function, I would expect nil return if MemBackend not set.

Reply:
Done